model/requirement-upload.js

```diff
@@ -116,6 +116,11 @@ module.exports = memoize(function (db) {
 			multiple: true
 		},
 
+		toWebServiceJSON: {
+			value: function (ignore) {
+				return this.document.toWebServiceJSON();
```

Comment:
Not really doable to provide owner, I think. I don't see what would qualify something for an owner. The closest I came by is such a parent object which is instance of Person class... but this is still really bad condition.